src/Search/Numeric.js

```diff
@@ -0,0 +1,115 @@
+import { useDebouncedCallback } from '@react-hookz/web'
+import { Input, Select } from '@rebass/forms/styled-components'
+import { FiSlash } from 'react-icons/fi'
+
+import { NO_UNIT_STRING } from '../App/adapter/lib/create'
+import { Flex, Box, Button } from '../Primitives'
+import { SEPARATE_CHAR, useFilters } from '../filters'
+import { JOIN_CHAR, useQueryParam } from '../router-utils'
+import FilterBox from './Filter'
+
+const onlyMax = (max, unit) =>
+  `${max + SEPARATE_CHAR}lte${unit ? SEPARATE_CHAR + unit : ''}`
+const onlyMin = (min, unit) =>
+  `${min + SEPARATE_CHAR}gte${unit ? SEPARATE_CHAR + unit : ''}`
+const both = (min, max, unit) =>
+  `${min + JOIN_CHAR + max + SEPARATE_CHAR}between${
+    unit ? SEPARATE_CHAR + unit : ''
+  }`
+
+function Numeric({ obj }) {
+  const { id, display, units = [] } = obj
+  const [defaultUnit, ...otherUnits] = units
```

Comment:
Good for now, but eventually, I think we'll have to have a separate `defaultUnit` key in the fitlerables in case the default unit is not the first in the array (though the first can still be the default when `defaultUnit` is not specified).